package-lock.json

```diff
@@ -9795,240 +9795,6 @@
         "node": ">=12"
       }
     },
-    "node_modules/@wordpress/block-editor/node_modules/@emotion/cache": {
```

Comment:
Please undo changes to this file. Looks like caused by `npm update`

Reply:
Oh yep, sorry about that. I thought I has stashed it.